_specs/ecip-1000.md

```diff
@@ -291,66 +284,7 @@ To avoid doubt: comments and status are unrelated metrics to judge an ECIP, and
-    License-Code: GPL-3.0+  # This refers to GPL v3.0 *or later*.
-
-In the event that the licensing for the text or code is too complicated to express with a simple list of alternatives, the list should instead be replaced with the single term "Complex". In all cases, details of the licensing terms must be provided in the Copyright section of the ECIP.
-
-ECIPs are not required to be *exclusively* licensed under approved terms, and may also be licensed under unacceptable licenses *in addition to* at least one acceptable license.
-In this case, only the acceptable license(s) should be listed in the License and License-Code headers.
-
-### Recommended licenses
-
-* Apache-2.0: [Apache License, version 2.0](http://www.apache.org/licenses/LICENSE-2.0)
-* BSD-2-Clause: [OSI-approved BSD 2-clause license](https://opensource.org/licenses/BSD-2-Clause)
-* BSD-3-Clause: [OSI-approved BSD 3-clause license](https://opensource.org/licenses/BSD-3-Clause)
-* CC0-1.0: [Creative Commons CC0 1.0 Universal](https://creativecommons.org/publicdomain/zero/1.0/)
-* GNU-All-Permissive: [GNU All-Permissive License](http://www.gnu.org/prep/maintain/html_node/License-Notices-for-Other-Files.html)
-
-In addition, it is recommended that literal code included in the ECIP be dual-licensed under the same license terms as the project it modifies. For example, literal code intended for Ethereum Classic Core would ideally be dual-licensed under the MIT license terms as well as one of the above with the rest of the ECIP text.
-
-### Not recommended, but acceptable licenses
-
-* BSL-1.0: [Boost Software License, version 1.0](http://www.boost.org/LICENSE_1_0.txt)
-* CC-BY-4.0: [Creative Commons Attribution 4.0 International](https://creativecommons.org/licenses/by/4.0/)
-* CC-BY-SA-4.0: [Creative Commons Attribution-ShareAlike 4.0 International](https://creativecommons.org/licenses/by-sa/4.0/)
-* MIT: [Expat/MIT/X11 license](https://opensource.org/licenses/MIT)
-* AGPL-3.0+: [GNU Affero General Public License (AGPL), version 3 or newer](http://www.gnu.org/licenses/agpl-3.0.en.html)
-* FDL-1.3: [GNU Free Documentation License, version 1.3](http://www.gnu.org/licenses/fdl-1.3.en.html)
-* GPL-2.0+: [GNU General Public License (GPL), version 2 or newer](http://www.gnu.org/licenses/old-licenses/gpl-2.0.en.html)
-* LGPL-2.1+: [GNU Lesser General Public License (LGPL), version 2.1 or newer](http://www.gnu.org/licenses/old-licenses/lgpl-2.1.en.html)
-
-## Rationale
-
-**Why are there software licenses included?**
-
-* Some ECIPs, especially consensus layer, may include literal code in the ECIP itself which may not be available under the exact license terms of the ECIP.
-* Despite this, not all software licenses would be acceptable for content included in ECIPs.
+You may not specify a license. Claims to any license other than the one specified [here](https://github.com/ethereumclassic/ECIPs/blob/master/LICENSE) are invalid.
```

Comment:
```suggestion
You may specify any license you want. However, in order for a proposal to be permanently accepted to this repository (eg. merged via a [Github Pull Request](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/about-pull-requests)), it must be associated with an [Apache 2 License](https://www.apache.org/licenses/LICENSE-2.0.html). 

For authors with alternative license preferences, it is suggested that you choose an even more lenient license (eg. [The Unlicense](https://choosealicense.com/licenses/unlicense/)) so that your work can be copied verbatim to this repository by any enterprising member of "the community" artistic, enterprising, or selfless enough to lay jealous claim to your ideas. In the case that this path is undesirable, you are free to pursue licensing and distribution of your ideas at will in any forum and any manner you wish; please only be advised that it will probably be more profitable to collect payment for your ideas and work _before_ actually sharing them with a bunch of pseudo-anonymous crypto-anarchists you met on the internet.  

By leaving this section blank, when and if your document's content is officially and technically included in this repository's `master` branch (no matter the labeled _Status_ of the document), the document will, along with its fore-running peers, fall under the Apache 2 License prescribed. 
```

Reply:
I disagree. It presents unnecessary challenges and risks to allow people to license the contents of this repo differently. 1 GPL ECIP and the whole repo is tainted.

Reply:
?

> You may specify any license you want. However, in order for the document content of a proposal to be permanently accepted to this repository (eg. merged via a [Github Pull Request](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/about-pull-requests)), it must be associated with an [Apache 2 License](https://www.apache.org/licenses/LICENSE-2.0.html).